py/tests/discover_and_run_tests.py

```diff
@@ -0,0 +1,53 @@
+import os
+import sys
+import unittest
+
+from absl import app
+from absl import flags
+from tests.colorful_tests import ColorfulTestRunner
+
+# Define command-line flags using absl
+FLAGS = flags.FLAGS
+_TYPE = flags.DEFINE_string(
+    "type", None,
```

Comment:
adding more precise name would be helpful in understanding the nature of the flag. i.e. test_type

Reply:
Updated by `_TEST_TYPE`  and `_FILE_PATTERN`
```python
_TEST_TYPE = flags.DEFINE_string(
    "type",
    None,
    "Specify the type of tests to run (e.g., 'functional', 'integration',"
    " 'performance',..).",
)
_FILE_PATTERN = flags.DEFINE_string(
    "pattern",
    "test_*.py",
    "Specify the file pattern to match (default is 'test_*.py').",
)
```